frontend/app/utils/search.ts

```
@@ -140,23 +168,22 @@
-    const startDate = rangeValues[0];
-    const endDate = rangeValues[1];
 
     return new InputJson(
       filters,
       rangeValue,
-      startDate,
-      endDate,
-      params.get(this.keyMap.sort) || 'startTs',
-      params.get(this.keyMap.order) || 'desc',
-      params.get(this.keyMap.eventsOrder) || 'then'
+      params.get(this.keyMap.sort) || DEFAULT_SORT,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)